site/content/en/docs/Installation/helm.md

```diff
@@ -94,6 +94,7 @@ The following tables lists the configurable parameters of the Open Match chart a
 | `open-match-core.pendingReleaseTimeout` | Defines the time before a ticket returns to the pool after it was fetched. | `1m` |
 | `open-match-core.assignedDeleteTimeout` | Time after a ticket has been assigned before it is automatically deleted. | `10m` |
 | `open-match-core.queryPageSize` | Maximum number of tickets to return on a single QueryTicketsResponse. | `10000` |
+| `open-match-core.backfillLockTimeout` | Defines the default time for a backfill lock should live. | `1m` |
```

Comment:
I suggest - Defines the time of keeping a lock on a single backfill.

Reply:
yes, that should be revised . Because the way it is here, seems that this is backfillTTL for cleanup backfill which is not.